### PR TITLE
Create directory structure for metric queries

### PR DESCRIPTION
**Pull request by KJLarson**
Progress on #62

### CODE REVIEW

File: src/config/contributors.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
```
Comment (rviscomi) [inline src/config/contributors.json]: This PR should be limited to the SQL directories only. Changes related to the contributors PR should be removed.

**Comment by rviscomi** [comment]
It's also good practice to include a short note in the top comment about the issue this PR is addressing. For example, in this case `Progress on #62`. Or for example if you're resolving an issue you can write `Fixes #62`. Could you edit your first comment with a note like this?

File: src/config/contributors.json (hunk 1 above)
Comment (KJLarson) [reply src/config/contributors.json]: Oh, yes. I was having some issues switching between branches, and I think I did some stuff in the wrong order. I think I figured out what I need to do for the future. As soon as I figure out how to remove the changes from the contributors PR, I will do so!

Comment (rviscomi) [reply src/config/contributors.json]: You should be able to straight up delete the files from within this branch and make a new commit that logs those deletions. They'll still exist in the other branch.

I think what might have happened was that you branched from that branch, which included all of its changes. Tip: I always sync/branch from master to ensure a clean branch.

Comment (rviscomi) [reply src/config/contributors.json]: `contributors.json` is still showing up in the PR. Can you delete it? Then this should be good to go.

**Pull request merged**